### PR TITLE
Add etcd operational guide

### PR DESCRIPTION
**Pull request by richardartoul**

### CODE REVIEW

File: docs/operational_guide/etcd.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# etcd
```
Comment (benraskin92) [inline docs/operational_guide/etcd.md]: Do we want to have recs on how many external etcd hosts one should have? is there some sort of formula based on the size of the m3db cluster?

File: docs/operational_guide/etcd.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+The M3 stack leverages M3 as a distributed key-value storage to:
```
Comment (benraskin92) [inline docs/operational_guide/etcd.md]: leverage `etcd` you mean?

File: docs/operational_guide/etcd.md (hunk 1 above)
Comment (schallert) [reply docs/operational_guide/etcd.md]: There's some etcd docs on the subject we can link to: https://github.com/etcd-io/etcd/blob/v3.3.11/Documentation/faq.md#what-is-failure-tolerance

File: docs/operational_guide/etcd.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        cacheDir: /var/lib/m3kv
+        etcdClusters:
+            - zone: embedded
+              endpoints:
+                  - http://external_etcd1:2379
+                  - http://external_etcd2:2379
+                  - http://external_etcd3:2379
+```
+
+**Note**: `M3DB` placements and namespaces are stored in `etcd` so if you want to switch to an external `etcd` cluster you'll need to recreate all your placements and namespaces.
```
Comment (schallert) [inline docs/operational_guide/etcd.md]: `etcdctl` has a feature to copy data from 1 cluster to another: https://github.com/etcd-io/etcd/blob/v3.3.11/etcdctl/doc/mirror_maker.md

Maybe we can link that here?

**Comment by richardartoul** [comment]
@schallert good call, done

**Pull request merged**